From a6c5a0c329fd1582fc94b51ba9c6f1c21abc0084 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Thu, 30 Jan 2025 22:04:41 +0100
Subject: [PATCH 8/8] Fedora local mod: integrate usbio-drivers within
 ipu6-drivers Makefile

Integrate usbio-drivers within ipu6-drivers Makefile.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 Makefile | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/Makefile b/Makefile
index 7ae3be835..637382d6d 100644
--- a/Makefile
+++ b/Makefile
@@ -114,6 +114,17 @@ ifeq ($(call version_lt,$(KERNEL_VERSION),$(KV_IVSC)),true)
 ccflags-y += -I$(src)/backport-include/drivers/misc/mei/
 endif
 
+# begin integrate usbio-drivers within ipu6-drivers Makefile
+obj-m += usbio.o
+usbio-y := drivers/mfd/usbio.o
+
+obj-m += gpio-usbio.o
+gpio-usbio-y := drivers/gpio/gpio-usbio.o
+
+obj-m += i2c-usbio.o
+i2c-usbio-y := drivers/i2c/busses/i2c-usbio.o
+# end integrate usbio-drivers within ipu6-drivers Makefile
+
 subdir-ccflags-y += -I$(src)/include/ \
 	-DCONFIG_VIDEO_V4L2_SUBDEV_API
 
-- 
2.47.1

